### PR TITLE
Added Label Component to Bit.Client.Web.BlazorUI

### PR DESCRIPTION
**Pull request by HamedMoghadasi**
Implement label component and Add to playground.

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Label/BitLabelTests.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Label
+{
+    [TestClass]
+    public class BitLabelTests: BunitTestContext
+    {
+        [TestMethod]
+        public void IsRequired_HasRequiredClass_True()
```
Comment (ysmoradi) [inline src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Label/BitLabelTests.cs:10]: Could you please test data for this, so we can test both required and optional labels at the same time?

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bit.Client.Web.BlazorUI.Components
+{
+    public partial class BitLabel
+    {
+        [Parameter] public string Text { get; set; }
+        [Parameter] public bool IsRequired { get; set; }
+        [Parameter] public string componentRef { get; set; }
```
Comment (ysmoradi) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs:14]: Could you please provide me a link that describes this approach? I mean string componentRef

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+        public string RequiredClass => IsRequired ? "required" : "";
```
Comment (ysmoradi) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs:16]: use string.Empty instead of ""

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs (hunk 2 above)
Comment (HamedMoghadasi) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs]: this property refers to "for" attribute of "label" element. 
Um, I think ,my naming wasn't good and made ambiguity, so I changed its name.

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs (hunk 3 above)
Comment (HamedMoghadasi) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/BitLabel.razor.cs]: 👌

**Review by ysmoradi — APPROVED** [review]

**Pull request merged**